erpnext_autoinstall/commands/__init__.py

```diff
@@ -164,6 +188,35 @@ def set_user_role_profile(username, role_profile):
     _set_user_role_profile(username, role_profile)
 
 
-commands = [list_users,
-            delete_user, add_user, set_user_password, set_user_roles,
-            set_user_role_profile]
+@click.command('get-user-api-secret', help="Generate API secret")
+@click.argument("username")
+@pass_context
+@connect_to_db
+def get_user_api_secret(username):
+    """Get user's secret api by username """
+    if frappe.db.exists('User', {'username': username}):
+        _get_user_api_secret(username)
+
+
+@click.command('get-user-api-key', help="Get API key")
+@click.argument("username")
+@pass_context
+@connect_to_db
+def get_user_api_key(username):
+    """Get user's api key by username"""
+    if frappe.db.exists("User", {"username": username}):
+        _get_user_api_key(username)
```

Comment:
If we wish to improve code coverage, we could move this inside the private function.

This would also allow to add unit tests for when user does not exist.